workshop/participant-information.md

```diff
@@ -10,7 +10,10 @@ We will meet in the Amenity Suite located on the lower level.
 Enter the building through the Center Lobby and take the elevator down one floor (labeled “LL”).
 The Amenity Suite is the first room you will see after exiting the elevator.
 Please note that Bala Plaza is an office park with multiple buildings.
-If you have trouble finding the building or the room on the first day of training, please use Slack to direct message `Jen O'Malley`.
+If you have trouble finding the building or the room on the first day of training, please use Slack to [direct message](../software-setup/slack-procedures.html#using-direct-messages-during-training) `Jen O'Malley`.
```

Comment:
Noting the title of the Slack Procedures page should be updated to remove "Virtual" since we are using it for both.

Maybe rename this subpage page to "local-participant-information"?

Reply:
Weird that you have to use html here, but jekyll is weird! 

Just as an alternative, in this case you don't need `..`since that just takes you out to the root of the site, which we can address directly by starting with `/`. Sometimes clearer. (But if you do this, do it all the `../` places that it makes sense.)
```suggestion
If you have trouble finding the building or the room on the first day of training, please use Slack to [direct message](/software-setup/slack-procedures.html#using-direct-messages-during-training) `Jen O'Malley`.
```

Reply:
noting this was the only spot, got it in https://github.com/AlexsLemonade/training-specific-template/pull/153/commits/f85fab27054cb877ae0be29ba4560071e850db37